Source/Schema.NET/core/WebPageElement.cs

```diff
@@ -15,5 +15,19 @@ public partial class WebPageElement : CreativeWork
         /// </summary>
         [DataMember(Name = "@type", Order = 1)]
         public override string Type => "WebPageElement";
+
+        /// <summary>
+        /// A CSS selector, e.g. of a &lt;a class="localLink" href="http://schema.org/SpeakableSpecification"&gt;SpeakableSpecification&lt;/a&gt; or &lt;a class="localLink" href="http://schema.org/WebPageElement"&gt;WebPageElement&lt;/a&gt;. In the latter case, multiple matches within a page can constitute a single conceptual "Web page element".
+        /// </summary>
+        [DataMember(Name = "cssSelector", Order = 206)]
+        [JsonConverter(typeof(ValuesConverter))]
+        public OneOrMany<CssSelectorType>? CssSelector { get; set; }
```

Comment:
I guess that you did not run the tool to generate these two properties. Maintaining this by hand is not really scalable due to the number of types. However, the tool project has some overrides that add extra properties that don't exist in schema.org but are defined in Google structured data.

Reply:
I actually ran the tool as is, and these came up automatically. I actually added those 2 files by hand, as these were not generated (they are basically strings)

Reply:
See https://github.com/RehanSaeed/Schema.NET/tree/master/Source/Schema.NET.Tool/Overrides for examples of how to add stuff like this.